wandb/cli.py

```diff
@@ -480,11 +480,18 @@ def pull(run, project, entity):
 
 @cli.command(context_settings=CONTEXT, help="Login to Weights & Biases")
 @click.argument("key", nargs=-1)
+@click.option("--host", default="https://api.wandb.ai", help="Login to a specific instance of W&B")
```

Comment:
This means that our on-prem users will have to specify host everytime they do a login rather than being able to rely on local or global settings files?

Reply:
I think host should be optional where it doesnt change anything if it wasnt specified.